### PR TITLE
Edited code to print log messages

### PR DESCRIPTION
**Pull request by elle-Bee**
Made changes in the code to log errors if there is no container to forward to. Also created a function GetLabel() which checks if there is a container to forward. This check happens in the beginning of the program to save time.

### CODE REVIEW

File: docker-lb/docker-handler/docker.go [hunk 1]
```diff
@@ -59,14 +60,9 @@ func (dh *DockerHandler) GetServiceInfo(containerID string) (*models.Server, err
 	labels := inspect.Config.Labels
 
 	// fetch label name
-	key, isFound := os.LookupEnv("LB_LABEL_NAME")
-	if !isFound {
-		key = "com.docker-lb.load-balance"
-		dh.Logger.Printf("LB_LABEL_NAME not found, falling back to %s", key)
-	}
-
+	key := "com.docker-lb.load-balance"
```
Comment (Shreyas220) [inline docker-lb/docker-handler/docker.go:63]: can you remove this 
we just added this to test something not needed anymore

File: docker-lb/docker-handler/docker.go [hunk 2]
```diff
@@ -59,14 +60,9 @@ func (dh *DockerHandler) GetServiceInfo(containerID string) (*models.Server, err
...
 	if _, ok := labels[key]; !ok {
```
Comment (Shreyas220) [inline docker-lb/docker-handler/docker.go:64]: change Key to `dh.Docker.Label`

**Review by Shreyas220 — CHANGES_REQUESTED** [review]
LGTM  :+1: 
just a few changes required 
#22 #21

File: docker-lb/docker-handler/docker.go (hunk 1 above)
Comment (elle-Bee) [reply docker-lb/docker-handler/docker.go:63]: Sure!

File: docker-lb/docker-handler/docker.go (hunk 2 above)
Comment (elle-Bee) [reply docker-lb/docker-handler/docker.go:64]: Should I change it to dh.DockerHandler.Label because dh.Docker does not contain Label in it ?

Comment (Shreyas220) [reply docker-lb/docker-handler/docker.go:64]: no 
its a variable

Comment (elle-Bee) [reply docker-lb/docker-handler/docker.go:64]: Ok I got confused. So it should be dh.DockerHandler.Label instead of dh.docker.Label right?

Comment (Shreyas220) [reply docker-lb/docker-handler/docker.go:64]: aah sorry 
it should be just   dh.Label

File: docker-lb/docker-handler/docker.go [hunk 3]
```diff
@@ -59,14 +60,9 @@ func (dh *DockerHandler) GetServiceInfo(containerID string) (*models.Server, err
 	labels := inspect.Config.Labels
 
 	// fetch label name
-	key, isFound := os.LookupEnv("LB_LABEL_NAME")
-	if !isFound {
-		key = "com.docker-lb.load-balance"
-		dh.Logger.Printf("LB_LABEL_NAME not found, falling back to %s", key)
-	}
-
+	key := "dh.Label"
```
Comment (Shreyas220) [inline docker-lb/docker-handler/docker.go:63]:
```suggestion

```

File: docker-lb/docker-handler/docker.go [hunk 4]
```diff
@@ -59,14 +60,9 @@ func (dh *DockerHandler) GetServiceInfo(containerID string) (*models.Server, err
...
 	if _, ok := labels[key]; !ok {
```
Comment (Shreyas220) [inline docker-lb/docker-handler/docker.go:64]:
```suggestion
	if _, ok := labels[dh.Label]; !ok {
```

**Review by Shreyas220 — CHANGES_REQUESTED** [review]

File: docker-lb/docker-handler/docker.go [hunk 5]
```diff
@@ -59,14 +60,9 @@ func (dh *DockerHandler) GetServiceInfo(containerID string) (*models.Server, err
...
-		return &models.Server{}, nil
+		return nil, nil
 	}
 
 	server.ServiceName = labels[key]
```
Comment (Shreyas220) [inline docker-lb/docker-handler/docker.go:68]:
```suggestion
	server.ServiceName = labels[dh.Label]
```

**Review by Shreyas220 — CHANGES_REQUESTED** [review]

File: docker-lb/docker-handler/docker.go (hunk 5 above)
Comment (elle-Bee) [reply docker-lb/docker-handler/docker.go]: Made the changes
Also in  `docker.go`, the key variable was redundant so deleted it too

**Review by Shreyas220 — APPROVED** [review]
Awesome!!
LGTM

**Pull request merged**